iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
> However, because #container has no size, setting the size on a descendant will also affect the size of #container. When you resize the browser, the list takes the new scroll target's offsetHeight which is incorrect anyways.

Why are you considering the new scroll target's `offsetHeight` incorrect? Or let me rephrase the question. Can an `iron-list` not be used as a "dumb" list instead of a "smart" list, such that all elements are always rendered, unless there is an explicit size set? This would make the default behavior more intuitive and it would match closer what the browser does in similar cases (as you stated in your comment above).